### PR TITLE
Exowear Pockets

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Saw it on bay and many other servers, thought it was cool, asked if anyone would like it, lotsa people said it would be cool. Works, tested multiple times.

## Why It's Good For The Game
I think having the option to use exowear for storage, besides the confusing one slot system, which is still left in for putting O2 tanks and the like in. Also, finally gives a use for the hazard vest.

## Changelog
:cl:
add: Two small pockets to exowear (Labcoats, Winter coats, Hazard Vests, etc.)
/:cl:

### CODE REVIEW

**Review by qwertyquerty — APPROVED** [review]

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -349,6 +349,7 @@
 	equip_delay_self = 50
 	strip_delay = 60
 	breakouttime = 3000
+	pocket_storage_component_path = FALSE
```
Comment (ike709) [inline code/modules/clothing/suits/miscellaneous.dm:352]: Are you sure this is supposed to be `FALSE` instead of `null`?

Comment (MarkSuckerberg) [reply code/modules/clothing/suits/miscellaneous.dm:352]: I mean, null would likely work as well, but the code seems to work as is

Comment (ike709) [reply code/modules/clothing/suits/miscellaneous.dm:352]: Did you check the View Runtimes verb for any runtimes when you were testing a subtype set to FALSE? If there wasn't any runtimes I'll sign off on this PR.

**Comment by MarkSuckerberg** [comment]
Ill check again but last time I checked there were none. I'll change it to null if a problem arises

**Comment by ike709** [comment]
Did you check again?

**Comment by MarkSuckerberg** [comment]
Sorry, been busy, have not gotten to it. Will when I get home

**Comment by MarkSuckerberg** [comment]
Yep, no runtimes, can still change it if you would prefer it as null. Sorry for taking forever

**Pull request merged**